<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMTrainSettingsWidget</class>
 <widget class="QWidget" name="QMTrainSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Schulungen</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Daten</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="leTrainFilter">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="tbTrainFilter">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Schulungen filtern</string>
             </property>
             <property name="text">
              <string>Filtern</string>
             </property>
             <property name="icon">
              <iconset resource="../rcs/icons.qrc">
               <normaloff>:/icons/icons/view-filter.svg</normaloff>:/icons/icons/view-filter.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QTableView" name="tvTrain">
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <attribute name="horizontalHeaderMinimumSectionSize">
              <number>200</number>
             </attribute>
             <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
              <bool>true</bool>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderDefaultSectionSize">
              <number>30</number>
             </attribute>
             <attribute name="verticalHeaderHighlightSections">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderMinimumSectionSize">
              <number>30</number>
             </attribute>
             <attribute name="verticalHeaderShowSortIndicator" stdset="0">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QToolButton" name="tbTrainAdd">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Hinzufügen einer neuen Schulung</string>
               </property>
               <property name="text">
                <string>Hinzufügen</string>
               </property>
               <property name="icon">
                <iconset resource="../rcs/icons.qrc">
                 <normaloff>:/icons/icons/list-add.svg</normaloff>:/icons/icons/list-add.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="tbTrainRemove">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Entfernen einer selektierten Schulung</string>
               </property>
               <property name="text">
                <string>Entfernen</string>
               </property>
               <property name="icon">
                <iconset resource="../rcs/icons.qrc">
                 <normaloff>:/icons/icons/list-remove.svg</normaloff>:/icons/icons/list-remove.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="tbTrainRevert">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Zurücksetzen aller Änderungen an der Tabelle</string>
               </property>
               <property name="text">
                <string>Zurücksetzen</string>
               </property>
               <property name="icon">
                <iconset resource="../rcs/icons.qrc">
                 <normaloff>:/icons/icons/edit-reset.svg</normaloff>:/icons/icons/edit-reset.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Gruppen</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTableView" name="tvTrainGroups">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>150</number>
           </attribute>
           <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
            <bool>true</bool>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>30</number>
           </attribute>
           <attribute name="verticalHeaderHighlightSections">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderMinimumSectionSize">
            <number>30</number>
           </attribute>
           <attribute name="verticalHeaderShowSortIndicator" stdset="0">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QToolButton" name="tbTrainGroupsAdd">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Hinzufügen einer neuen Schulungsgruppe</string>
             </property>
             <property name="text">
              <string>Hinzufügen</string>
             </property>
             <property name="icon">
              <iconset resource="../rcs/icons.qrc">
               <normaloff>:/icons/icons/list-add.svg</normaloff>:/icons/icons/list-add.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="tbTrainGroupsRemove">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Entfernen einer selektierten Schulungsgruppe</string>
             </property>
             <property name="text">
              <string>Entfernen</string>
             </property>
             <property name="icon">
              <iconset resource="../rcs/icons.qrc">
               <normaloff>:/icons/icons/list-remove.svg</normaloff>:/icons/icons/list-remove.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="tbTrainGroupsRevert">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Zurücksetzen aller Änderungen an der Tabelle</string>
             </property>
             <property name="text">
              <string>Zurücksetzen</string>
             </property>
             <property name="icon">
              <iconset resource="../rcs/icons.qrc">
               <normaloff>:/icons/icons/edit-reset.svg</normaloff>:/icons/icons/edit-reset.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Status</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTableView" name="tvTrainState">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>150</number>
           </attribute>
           <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
            <bool>true</bool>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>30</number>
           </attribute>
           <attribute name="verticalHeaderHighlightSections">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderMinimumSectionSize">
            <number>30</number>
           </attribute>
           <attribute name="verticalHeaderShowSortIndicator" stdset="0">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QToolButton" name="tbTrainStateAdd">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Hinzufügen einer neuen Schulungsgruppe</string>
             </property>
             <property name="text">
              <string>Hinzufügen</string>
             </property>
             <property name="icon">
              <iconset resource="../rcs/icons.qrc">
               <normaloff>:/icons/icons/list-add.svg</normaloff>:/icons/icons/list-add.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="tbTrainStateRemove">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Entfernen einer selektierten Schulungsgruppe</string>
             </property>
             <property name="text">
              <string>Entfernen</string>
             </property>
             <property name="icon">
              <iconset resource="../rcs/icons.qrc">
               <normaloff>:/icons/icons/list-remove.svg</normaloff>:/icons/icons/list-remove.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="tbTrainStateRevert">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Zurücksetzen aller Änderungen an der Tabelle</string>
             </property>
             <property name="text">
              <string>Zurücksetzen</string>
             </property>
             <property name="icon">
              <iconset resource="../rcs/icons.qrc">
               <normaloff>:/icons/icons/edit-reset.svg</normaloff>:/icons/icons/edit-reset.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../rcs/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>leTrainFilter</sender>
   <signal>editingFinished()</signal>
   <receiver>QMTrainSettingsWidget</receiver>
   <slot>filterTrain()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbTrainGroupsAdd</sender>
   <signal>clicked()</signal>
   <receiver>QMTrainSettingsWidget</receiver>
   <slot>addTrainGroups()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>836</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbTrainGroupsRemove</sender>
   <signal>clicked()</signal>
   <receiver>QMTrainSettingsWidget</receiver>
   <slot>removeTrainGroups()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>836</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbTrainGroupsRevert</sender>
   <signal>clicked()</signal>
   <receiver>QMTrainSettingsWidget</receiver>
   <slot>revertTrainGroups()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>836</x>
     <y>520</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbTrainFilter</sender>
   <signal>clicked()</signal>
   <receiver>QMTrainSettingsWidget</receiver>
   <slot>filterTrain()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>836</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbTrainAdd</sender>
   <signal>clicked()</signal>
   <receiver>QMTrainSettingsWidget</receiver>
   <slot>addTrain()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>835</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbTrainRemove</sender>
   <signal>clicked()</signal>
   <receiver>QMTrainSettingsWidget</receiver>
   <slot>removeTrain()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>835</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbTrainRevert</sender>
   <signal>clicked()</signal>
   <receiver>QMTrainSettingsWidget</receiver>
   <slot>revertTrain()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>835</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbTrainStateAdd</sender>
   <signal>clicked()</signal>
   <receiver>QMTrainSettingsWidget</receiver>
   <slot>addTrainState()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>834</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbTrainStateRemove</sender>
   <signal>clicked()</signal>
   <receiver>QMTrainSettingsWidget</receiver>
   <slot>removeTrainState()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>834</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbTrainStateRevert</sender>
   <signal>clicked()</signal>
   <receiver>QMTrainSettingsWidget</receiver>
   <slot>revertTrainStates()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>834</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addTrain()</slot>
  <slot>removeTrain()</slot>
  <slot>revertTrain()</slot>
  <slot>addTrainGroups()</slot>
  <slot>removeTrainGroups()</slot>
  <slot>revertTrainGroups()</slot>
  <slot>filterTrain()</slot>
  <slot>addTrainState()</slot>
  <slot>removeTrainState()</slot>
  <slot>revertTrainStates()</slot>
 </slots>
</ui>
